### DISCUSSION TITLE
Tutanota email settings don't work - cannot set 'send from'/'reply to' email

### DISCUSSION THREAD

**Discussion by DianneCT** [tutao/tutanota#6396] [Bugs]
I only have Tutanota as an email client (paid account) in the back end of my main email address which comes through my web domain (but does not store emails there). I followed Tuta's instructions for setting my own 'send from' and 'reply to' email addresses, NOT with the @tutanota.com domain on them, because my recipients MUST see my emails as being sent from my main email domain, NOT my tutanota address. But it keeps sending from my tutanota address, and it is screwing up all my 2FAs and KYC identification processes with new accounts, such as banks, employers, and currency exchanges. Driving me crazy! If no solution to this, I will have to just ditch Tutanota as it is wrecking my business life.

**Comment by charlag** [comment]
Hi, I would recommend contacting support, they can look at your case more closely

**Comment by DianneCT** [comment]
How do you even contact Support?
____________________________________
Dianne Trussell B.Sc. Hons

Researchgate <https://www.researchgate.net/profile/Dianne_Trussell>
Linkedin <https://au.linkedin.com/in/diannetrussell>

**Reply by charlag** [reply]
There's a question mark button next to the folder list, you type your query there and you should see a button to write email to support
